pkg/app/piped/platformprovider/lambda/client.go

```diff
@@ -133,6 +133,12 @@ func (c *client) CreateFunction(ctx context.Context, fm FunctionManifest) error
 			ImageUri: aws.String(fm.Spec.ImageURI),
 		}
 	}
+	if fm.Spec.VPCConfig != nil {
+		input.VpcConfig = &types.VpcConfig{
+			SecurityGroupIds: fm.Spec.VPCConfig.SecurityGroupIDs,
+			SubnetIds:        fm.Spec.VPCConfig.SubnetIDs,
+		}
+	}
```

Comment:
Could you move this block to L128 or after L152, I want to keep "packing using container image" and "Zip on S3" block near by since they mean to same thing (code of the function).

Reply:
You are right. I fix.